### PR TITLE
Remove PyICU and fix paths to requirements in setup.py

### PR DESCRIPTION
**Pull request by paveldedik**

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -1,13 +1,11 @@
-# SHA1:728467968860943ba7eecd70ca33a9034bbc00ad
 #
-# This file is autogenerated by pip-compile-multi
```
Comment (aexvir) [inline requirements/base.txt:3]: You didn't use `pip-compile-multi`?

Comment (paveldedik) [reply requirements/base.txt:3]: Didn't, just about to fix that, forgot we use it now.

File: setup.py [hunk 2]
```diff
@@ -2,10 +2,10 @@
 
 from setuptools import find_packages, setup
 
-with open("base.txt") as f:
```
Comment (aexvir) [inline setup.py:5]: Hmm... we don't keep the `requirements` folder when copying to the Docker image, that's why it was changed here

Comment (paveldedik) [reply setup.py:5]: In Dockerfile, we use `pip install -r ...`, but e.g. `tox` is installing via `setup.py` and it was broken. This fixes it, or is it a bad fix?

Comment (aexvir) [reply setup.py:5]: We also do `pip install -e .`, but from what I can see in the CI it works fine. So let's merge it

**Review by aexvir — APPROVED** [review]

**Pull request merged**